front/components/timeline/TimlineStep.tsx

```diff
@@ -1,14 +1,13 @@
 import type { FC } from "react";
 import * as React from "react";
-import type { ImageSourcePropType } from "react-native";
-import { Image, StyleSheet } from "react-native";
+import { StyleSheet } from "react-native";
 
 import Colors from "../../constants/Colors";
 import { Text, View } from "../Themed";
 
 interface TimelineStepProps {
   title: string;
-  icon: ImageSourcePropType;
+  icon: React.ReactElement;
```

Comment:
Préférer `React.ReactNode` comme type : https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement

Reply:
Dans le code précédent, icon était une string, un type qui est dans ReactNode. L'implem peut changer souvent entre un composant React et une string (et même undefined), donc autant garder ReactNode qui englobe tout ça